web/src/search/helpers.test.tsx

```diff
@@ -86,4 +96,81 @@ describe('search/helpers', () => {
             )
         })
     })
+
+    describe('suggestions', () => {
+        const filterQuery = 'test r test'
+
+        const getArchivedSuggestions = () => filterSearchSuggestions('archived:', 9, searchFilterSuggestions)
+        const getFilterSuggestionStartingWithR = () => filterSearchSuggestions(filterQuery, 6, searchFilterSuggestions)
+
+        describe('filterSearchSuggestions()', () => {
+            test('filters suggestions for filters starting with "r"', () => {
+                const filtersStartingWithR = Object.keys(searchFilterSuggestions).filter(startsWith('r'))
+                expect(map(getFilterSuggestionStartingWithR(), 'value')).toEqual(
+                    expect.arrayContaining(filtersStartingWithR)
+                )
+            })
+
+            test('filters suggestions for filter aliases', () => {
+                forEach(filterAliases, (filter: string, alias: string) => {
```

Comment:
We prefer `for of` over `forEach()` because it can be stepped through with a debugger and supports `continue` and `break`, plus we should use the platform where possible instead of a library
```suggestion
                for (const [filter, alias] of Object.entries(filterAliases)) {
```